features/guidance/destination-signs.feature

```diff
@@ -0,0 +1,38 @@
+@routing @guidance
+Feature: Destination Signs
+
+    Background:
+        Given the profile "car"
+
+    Scenario: Car - route name assembly with destination signs
+        Given the node map
+          | a | b |
+          | c | d |
+          | e | f |
+          | g | h |
+          | i | j |
+          | k | l |
+          | m | n |
+          | o | p |
+
+        And the ways
+          | nodes | name | ref | destination    | destination:ref | oneway | #                                    |
+          | ab    | AB   | E1  |                |                 | yes    |                                      |
+          | cd    | CD   |     | Berlin         |                 | yes    |                                      |
+          | ef    | EF   |     | Berlin         | A1              | yes    |                                      |
+          | gh    |      |     | Berlin         | A1              | yes    |                                      |
+          | ij    |      |     | Berlin         |                 | yes    |                                      |
+          | kl    | KL   | E1  | Berlin         | A1              | yes    |                                      |
+          | mn    | MN   |     | Berlin;Hamburg | A1;A2           | yes    |                                      |
+          | op    | OP   |     | Berlin;Hamburg | A1;A2           | no     | mis-tagged destination: not a oneway |
+
```

Comment:
Yep that combination is covered by the lib function but not in the tests --- thanks, will add a cucumber test for this.